packages/modules/src/networkAnalysisDataVis/components/NetworkAnalysisVisualization.styles.ts

```diff
@@ -1,6 +1,28 @@
 import { makeStyles, Theme } from '@material-ui/core/styles'
 
-export default makeStyles((theme: Theme) => ({
+/**
+ * TODO: this is already declared in `client-ui`. Consider
+ * moving theme handling to a separate package that can be referenced
+ * here.
+ */
+interface CustomTheme extends Theme {
```

Comment:
Was this necessary here?

Reply:
Because it is declared in `client-ui` package which is not a dependency of the modules anymore. All theme related files need to be moved into a separate package later, then this code can be removed.